multimodal/setup.py

```diff
@@ -20,43 +20,41 @@
 submodule = "multimodal"
 install_requires = [
     # version ranges added in ag.get_dependency_version_ranges()
-    "numpy",
-    "scipy",
-    "pandas",
-    "scikit-learn",
-    "Pillow",
-    "tqdm",
+    "numpy",  # version range defined in `core/_setup_utils.py`
+    "scipy",  # version range defined in `core/_setup_utils.py`
+    "pandas",  # version range defined in `core/_setup_utils.py`
+    "scikit-learn",  # version range defined in `core/_setup_utils.py`
+    "Pillow",  # version range defined in `core/_setup_utils.py`
+    "tqdm",  # version range defined in `core/_setup_utils.py`
     "boto3",
     "requests",
-    "jsonschema<=4.8.0",
-    "seqeval<=1.2.2",
-    "evaluate<=0.3.0",
-    "accelerate>=0.9,<0.16",
+    "jsonschema<4.18",
+    "seqeval<1.2.3",
```

Comment:
Unless you expect patch versions to be breaking, I'd suggest
```suggestion
    "seqeval<1.3",
```

Reply:
Thanks. Revised.